app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person_data["name"], person_data["age"])
+                   for person_data in people]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = (
+                Person.people[person["wife"]]
+            )
+        elif person.get("husband"):
+            Person.people[person["name"]].husband = (
+                Person.people[person["husband"]]
```

Comment:
Similar to the previous comment, if `person["husband"]` does not exist in `Person.people`, this will raise a `KeyError`. Ensure the spouse exists before attempting to assign it.